include/exadg/incompressible_navier_stokes/user_interface/parameters.h

```diff
@@ -87,6 +97,9 @@ class Parameters
   void
   print_parameters_turbulence(dealii::ConditionalOStream const & pcout) const;
 
+  void
+  print_parameters_generalized_newtonian(dealii::ConditionalOStream const & pcout) const;
```

Comment:
Maybe unify to `print_parameters_variable_viscosity()`,  or skip member functions entirely if realized as suggested here https://github.com/exadg/exadg/pull/381/files#r1144984037

Reply:
We should introduce functions like `turbulence_model_data.print(pcout)` and `generalized_newtonian_model_data.print(pcout)`, so that we can skip these functions in `IncNS::Parameters`. We currently have done this for some parameter structs in ExaDG, but not consistently for all. I think the present PR would be a good chance to make the code better in this regard (of course, only for turbulence model data and the new generalized newtonian model data). i.e. this would improve our code in terms of the "single responsibility" design principle.

Reply:
Good idea :)
I adapted `viscosity_model_data` accordingly.
Everything in one place ... nice!